### ISSUE TITLE
crash on "hidden" unwanted screen

### ISSUE THREAD

**Issue by fungos** [nextcloud/android#7887]
### Steps to reproduce
1. Open NextCloud sync (without any account set)
2. In the connection screen, press Android back button two times - the timing between both presses is hard, try to sync with the left-to-right sliding animation, pressing the back button for the second time near the end of the animation.
3. If successful, a white screen will appear, looking like this:
![20210123_084627](https://user-images.githubusercontent.com/151544/105579965-90ee3400-5d57-11eb-91a0-dfc53c9ffa38.jpg)

4. In the above screen, if you press the + button, a crash occurs:
```
java.lang.NullPointerException: Attempt to invoke virtual method 'com.owncloud.android.lib.resources.status.CapabilityBooleanType com.owncloud.android.lib.resources.status.OCCapability.getRichDocuments()' on a null object reference
	at com.owncloud.android.ui.fragment.OCFileListBottomSheetDialog.onCreate(OCFileListBottomSheetDialog.java:128)
	at android.app.Dialog.dispatchOnCreate(Dialog.java:494)
	at android.app.Dialog.show(Dialog.java:342)
	at com.owncloud.android.ui.fragment.OCFileListFragment.lambda$registerFabListener$2$OCFileListFragment(OCFileListFragment.java:442)
	at com.owncloud.android.ui.fragment.-$$Lambda$OCFileListFragment$aYkoWPoxzAOahKL5AVjT06NhcnM.onClick(Unknown Source:4)
	at android.view.View.performClick(View.java:6897)
	at android.view.View$PerformClick.run(View.java:26104)
	at android.os.Handler.handleCallback(Handler.java:789)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6944)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
```

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140190
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: samsung
Device: heroltebmc
Model: SM-G930W8
Id: R16NW
Product: heroltebmc

************ FIRMWARE ************
SDK: 26
Release: 8.0.0
Incremental: G930W8VLS7CTB2

**Comment by AndyScherzinger** [comment]
@fungos does this also happen with the latest stable? You are running 3.14.1 while 3.14.3 is available.

**Comment by fungos** [comment]
The 3.14.3 is not on F-Droid yet.

**Comment by fungos** [comment]
Although there is a 3.15.0RC available, I tried it and it didn't crash in the + button, but crashed trying to create a new folder from there:

```
************ CAUSE OF ERROR ************

java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.datamodel.OCFile.getDecryptedRemotePath()' on a null object reference
	at com.owncloud.android.ui.dialog.CreateFolderDialogFragment.onClick(CreateFolderDialogFragment.java:139)
	at androidx.appcompat.app.AlertController$ButtonHandler.handleMessage(AlertController.java:167)
	at android.os.Handler.dispatchMessage(Handler.java:105)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6944)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30150051
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: samsung
Device: heroltebmc
Model: SM-G930W8
Id: R16NW
Product: heroltebmc

************ FIRMWARE ************
SDK: 26
Release: 8.0.0
Incremental: G930W8VLS7CTB2
```

**Comment by tobiasKaminsky** [comment]
This crashes as there is no server which can be used to create a file on it.
Obviously this should not be shown at all when there is no account.

I tried to reproduce it, but failed. Looking into code I also do not directly see when/how this could occur.

@fungos can you maybe create a video, so that I can see when it happens?

**Comment by fungos** [comment]
Here a quick video I did https://anonymousfiles.io/2nO5026J/

**Comment by tobiasKaminsky** [comment]
Thanks for the video.
It seems you can trigger this somehow reliable, while I still fail to do so…

This is not a serious bug, but still needs to be fixed. 
Thanks for reporting :+1:

**Comment by fungos** [comment]
Once you get the right timing, it is really easy to reproduce. I agree it is not serious.

**Comment by joshtrichards** [comment]
Fixed in #10384 
Release in v3.21.x
